src/open_inwoner/js/components/upload-document/file-input-errors.js

```diff
@@ -0,0 +1,17 @@
+const documentUpload = document.getElementById('formOpenUpload')
+const uploadError = document.querySelectorAll('.notifications')
+// get info
+const getFormInfo = document.querySelectorAll('.form__control--info')
+
+// if errors are present, scroll and trigger opened state
+if (uploadError.length > 1) {
+  console.log(uploadError)
+  documentUpload.scrollIntoView({
+    block: 'center',
+    behavior: 'smooth',
+  })
+  documentUpload.classList.add('upload--open')
+  for (let i = 0, max = getFormInfo.length; i < max; i++) {
```

Comment:
Can't we use only variable "i" here?Why do we need "max" as well?

Reply:
My guess is to save the limit condition (because `.length` is a calculated property)

But `for-i` loops are terrible, it is much better preferred to use `.forEach()` on the queryselector result, something like:

``
getFormInfo.forEach(elem => elem.style.display = 'none')
``

(a `for-of` loop would be nice but I don't know if that works with every weird JS itearble thing)